### PR TITLE
Settings Component & General Style Updates

### PR DESCRIPTION
**Pull request by livfwds**
**readers note** *any file where I removed a div and replaced with a span looks like I randomly added and deleted lines for some reason, but the only change was div -> span*

- all static settings elements are now rendered as React components
- the settingsPersonalInfo component is fully functioning (except for updating avatarUrl & password*)
- did a few general style updates

* updating the password will require us running it through bcrypt again, so I've made that an "if we have time" goal

@inklin @ggrochow

### CODE REVIEW

File: src/actions/account.js [hunk 1]
```diff
@@ -23,3 +23,14 @@ export function loadError(){
...
 
+export function setPersonalInfo(data) {
+  return {
+    type: AccountConstants.SET_PERSONAL_INFO,
+    firstName: data.firstName,
+    lastName: data.lastName,
+    email: data.email,
+    avatarUrl: data.avatarUrl,
+    userName: data.username,
+    password: data.password
```
Comment (ggrochow) [inline src/actions/account.js]: is this something we actually want to store / know about?

File: src/components/settings-personal-info.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  }
+
+  sendFormData = () => {
+    var formData = {
+      firstName: this.refs.firstName.value,
+      lastName: this.refs.lastName.value,
+      email: this.refs.email.value,
+      username: this.refs.username.value,
+      password: this.refs.password.value,
+      avatarUrl: `${this.props.currentUser.get('avatarUrl')}`,
```
Comment (ggrochow) [inline src/components/settings-personal-info.js]: you dont need a template string here as it should return a string, also assuming this is placeholder until we get imageuploads working for this

File: src/reducers/current-user.js [hunk 3]
```diff
@@ -29,6 +29,19 @@ function loadSuccess(state, action){
   })
 }
 
+function setPersonalInfo(state, action) {
```
Comment (ggrochow) [inline src/reducers/current-user.js]: it doesn't even look like this function is getting used?
setPersonalInfo in general that is

Comment (livfwds) [inline src/reducers/current-user.js]: line 22 of components/settings-personal-info
`this.props.dispatch(setPersonalInfo(formData));`

using it the same way you're using loadSuccess on line 20 of components/navbar I think?
`this.props.dispatch(loadSuccess(payload.data))`

File: src/components/settings-personal-info.js (hunk 2 above)
Comment (livfwds) [inline src/components/settings-personal-info.js]: ok... but yeah, it's just a placeholder for now

File: src/actions/account.js (hunk 1 above)
Comment (livfwds) [inline src/actions/account.js]: mmm... that's a good call... no...  was just matching everything I was pulling in with the form.

**Comment by ggrochow** [comment]
Im going to let @inklin merge this in when you have a sec to look at it

**Comment by inklin** [comment]
Looks great, way to go Em!

**Pull request merged**